### PR TITLE
prow/pluggins/*: Clarify where commands can be called

### PR DESCRIPTION
**Pull request by LiliC**
The commands meow, pony or woof can be called from a PR as well and it
works in the same way as in an issue, this makes that more clear to the
user.

### CODE REVIEW

**Comment by LiliC** [comment]
/meow
/pony
/woof

Testing :)

File: prow/plugins/dog/dog.go [hunk 1]
```diff
@@ -55,11 +55,11 @@ func init() {
 func helpProvider(config *plugins.Configuration, enabledRepos []string) (*pluginhelp.PluginHelp, error) {
 	// The Config field is omitted because this plugin is not configurable.
 	pluginHelp := &pluginhelp.PluginHelp{
-		Description: "The dog plugin adds a (famous.)dog image to an issue in response to the `/woof` command.",
+		Description: "The dog plugin adds a (famous.)dog image to an issue or PR in response to the `/woof` command.",
```
[the remark this answers is not in the document]
Comment (LiliC) [reply prow/plugins/dog/dog.go]: 🤷‍♀️ :)

[the remark this answers is not in the document]
Comment (ibrasho) [reply prow/plugins/dog/dog.go]: I believe it referring to a [specific](https://storage.googleapis.com/this-is-fine-images/this_is_fine.png) [famous](https://storage.googleapis.com/this-is-fine-images/this_is_not_fine.png) [dog](https://storage.googleapis.com/this-is-fine-images/this_is_unbearable.jpg). 😉

**Comment by ibrasho** [comment]
/lgtm

I remember seeing several other places that could use similar clarification.

Idea: Can we update commands to specify their contexts as part of the definition? I think standardizing this part is helpful. (Now I think about it, several aspects of defining commands can also be standardized).

**Comment by stevekuznetsov** [comment]
/lgtm

Thanks for the update @LiliC  :)
@ibrasho yes, we could potentially add those to the help framework.

**Comment by fejta** [comment]
Thanks for the contribution!
/pony party

**Comment by Katharine** [comment]
> Couldn't find a pony matching that query.

Huh, I wonder why it claimed it couldn't find a pony matching _nothing_. regex bug maybe?

**Pull request merged**